src/legacy/core_plugins/markdown_vis/public/markdown_vis.js

```diff
@@ -30,12 +30,10 @@ import { DefaultEditorSize } from 'ui/vis/editor_size';
 // register the provider with the visTypes registry so that other know it exists
 VisTypesRegistryProvider.register(MarkdownVisProvider);
 
-function MarkdownVisProvider(Private) {
-  const VisFactory = Private(VisFactoryProvider);
-
+function MarkdownVisProvider() {
```

Comment:
Could we remove the word provider from this function?
I also recall we had a lint rule that doesn't approve of functions starting with upper camel case.

Reply:
in this PR i am not removing the provider from markdown, will do this in a follow up here https://github.com/elastic/kibana/pull/38767